CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree iOS Visa Checkout SDK Release Notes
 
+## unreleased
+* Breaking changes
+  * Require `braintree_ios` major version v5
```

Comment:
The only breaking change from v5 that will affect VisaCheckout merchants is that the nonce no longer has a `description` property. I didn't think that was worth calling out here, since that change comes from `braintree_ios` and is listed in that CHANGELOG - open to if folks have a different opinion though!

Reply:
That seems reasonable to me